src/components/layout/header.tsx

```diff
@@ -62,15 +74,60 @@ const HeaderContent = styled.div`
   `)}
 `;
 
-const Header: React.FC<Props> = ({ isMenuOpen, setIsMenuOpen }) => (
-  <StyledHeader>
-    <HeaderContent>
-      <Link to="/" style={{ color: 'inherit', textDecoration: 'none' }}>
-        <Title type="navTitle" title="12062020" />
-      </Link>
-      <Burger isMenuOpen={isMenuOpen} setIsMenuOpen={setIsMenuOpen} />
-    </HeaderContent>
-  </StyledHeader>
-);
+const RightMenu = styled(Flex)`
+  > *:not(:last-child) {
+    margin-right: 25px;
+  }
+`;
+
+const HeaderMenuLink = styled(Link)`
+  color: ${colors.DefaultFontColor};
+  text-decoration: none;
+`;
+
+const Header: React.FC<Props> = ({ isMenuOpen, setIsMenuOpen }) => {
+  const { menu } = useStaticQuery<{
+    menu: MenuQuery;
+  }>(query);
+
+  return (
+    <StyledHeader>
+      <HeaderContent>
+        <Link to="/" style={{ color: 'inherit', textDecoration: 'none' }}>
+          <Title type="navTitle" title="12062020" />
+        </Link>
+        <RightMenu flexDirection="row">
+          <ScreenClass
```

Comment:
Just out of curiosity why did you implement the screen switch with `Screen` instead of media queries?

Reply:
@mtauer it is also removed from DOM like this, so we don't have top care of setting `aria-hidden`, ignoring it from DOM, etc....

ScreenClass uses the same breakpoints we use for media queries, so it should be consistent